ci/cpu/prebuild.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+#Upload cuspatial once per CUDA
+if [[ "$CUDA" == "9.2" ]]; then
+    export UPLOAD_CUSPATIAL=1
+else
+    export UPLOAD_CUSPATIAL=0
+fi
+
+#Upload libcuspatial once per PYTHON
```

Comment:
Only upload on Python 3.6 so it uploads on both CUDA 9.2 and CUDA 10.0
```suggestion
#Upload libcuspatial once per CUDA
```